### DISCUSSION TITLE
[BUG] Hi guys , im getting trouble loading the wallet , any suggestions plzz

### DISCUSSION THREAD

**Discussion by gnikhil002** [Chia-Network/chia-blockchain#9146] [Support]
![chia glitch](https://user-images.githubusercontent.com/93718968/140323487-872ea4a5-6361-41df-8a4b-bb763a4bb95a.PNG)

**Comment by lsherring** [comment]
Deleter your wallet DBase and let it resynch....You will still farm but the wallet should re-synch with a full node in a short amount of time.  Look at your error logs.  If it cannot connect to 127.0.0.1, that is your issue.

**Reply by gnikhil002** [reply]
Sure, I’ll  try doing that and will get back....
Thank you

**Reply by gnikhil002** [reply]
._debug.lock is shown in the log, do I have to delete that as well?

**Comment by grobalt** [comment]
Same for me after update from 1.2.7 to 1.2.11, needed to kill the wallet database :(
